velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -35,6 +35,39 @@ class ContainerRowSerdeTest : public testing::Test,
     return position;
   }
 
+  std::vector<std::shared_ptr<ByteStream>> serialize(
```

Comment:
This method can be simplified to just serialize the data and return a list of positions. No need to create and return ByteStreams. These can be created on the fly from the positions in testCompareWithNulls. I think this would also remove the need for rewindByteStreams method.

Reply:
Thanks for your suggestion, It is much cleaner now.